### PR TITLE
add client descriptions

### LINKED ISSUE #312: Create content for client highlights
We need brief content to highlight client work on the website. It should be simple, one paragraph short descriptions of the work that was made for them.

image

### PR DESCRIPTION
**Pull request by marcoow**
This adds descriptions for all of the clients that we list on the work page and a brief summary of what we did for them. I didn't add the ones we have success stories for anyway as that seemed repetitive to me.

closes #312

### CODE REVIEW

**Comment by marcoow** [comment]
@mrmrcoleman can you review this as well and check whether the wording makes sense? See preview here: https://deploy-preview-361--gallant-hawking-42050a.netlify.com/work

File: src/ui/styles/blocks/grid-block.block.css [hunk 1]
```diff
@@ -15,7 +15,6 @@
 
 .item {
   width: 19em;
-  height: 15em;
```
Comment (marcoow) [inline src/ui/styles/blocks/grid-block.block.css:18]: This of course affects and likely breaks other parts of the page as well but defining an arbitrary height in a generic CSS block seems wrong anyway?

Comment (pichfl) [reply src/ui/styles/blocks/grid-block.block.css:18]: Given that it's specified in `em` is a sign that it's something I haven't updated to the new system yet, so removing the value should be fine. We should consider using CSS grids for this specific layout anyway.

Comment (marcoow) [reply src/ui/styles/blocks/grid-block.block.css:18]: > We should consider using CSS grids for this specific layout anyway.

do you want me to change anything?

Comment (pichfl) [reply src/ui/styles/blocks/grid-block.block.css:18]: I'll be taking care of this as part of #367

**Comment by pichfl** [comment]
@marcoow can you rebase this PR?

**Comment by marcoow** [comment]
@pichfl: rebased; I'd like for @mrmrcoleman to have a quick look at this as well before merging though.

**Comment by mrmrcoleman** [comment]
"We have a track record of successful projects for international clients" sounds odd for some reason, perhaps because it sounds like only some of them are successful.

For the title I would instead use "A proven track record of successful projects for international clients."

**Comment by mrmrcoleman** [comment]
The text "Read about Trainline" is obscured on my screen.

Screen Shot 2019-04-30 at 5 36 54 PM

**Comment by mrmrcoleman** [comment]
Text for Timify is also obscured and shows the trainline logo
Screen Shot 2019-04-30 at 5 38 53 PM

**Comment by mrmrcoleman** [comment]
For cardstack I would change to this:

"Cardstack builds the experience layer for a decentralized internet. simplabs helped them complete their Card UI system and validate its abilities in various prototype projects, identifying and fixing some issues in their core framework along the way."

**Comment by mrmrcoleman** [comment]
Generali:

"Generali approached simplabs when they were looking for support with an internal Ember.js project. We guided their team during the course of the project by teaching and establishing best practices until the project's successful completion."

**Comment by mrmrcoleman** [comment]
xbAV:

"xbAV makes pensions accessible for employees, employers and agents. They approached simplabs when they were looking for support releasing a number of critical features. Our technology experts joined xbAV's internal team, increasing the available workforce while boosting their expertise."

**Comment by mrmrcoleman** [comment]
Experteer:

"Experteer is Europe's leading executive career service. simplabs has supported them in various ways over the years, building custom web apps, reviewing their code as well as providing architecture and process consulting."

File: src/ui/components/PageWork/template.hbs [hunk 2]
```diff
@@ -35,26 +35,106 @@
...
           </p>
         </div>
         <div class="grid-block.item">
           <div class="grid-block.image-wrapper">
-            <img class="grid-block.image" src="/assets/images/logos/timify.svg" />
+            <img class="grid-block.image" src="/assets/images/logos/cardstack.svg" />
           </div>
           <p class="grid-block.text">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Cardstack builds the experience layer for a decentralized internet. simplabs helped them completing their Card UI system and validate its abilities in various prototype projects, identifying and fixing some issues in their core framework along the way.
```
Comment (mrmrcoleman) [inline src/ui/components/PageWork/template.hbs]: Cardstack builds the experience layer for a decentralized internet. simplabs helped them complete their Card UI system and validate its abilities in various prototype projects, identifying and fixing some issues in their core framework along the way.

File: src/ui/components/PageWork/template.hbs [hunk 3]
```diff
@@ -35,26 +35,106 @@
...
           </p>
         </div>
         <div class="grid-block.item">
           <div class="grid-block.image-wrapper">
-            <img class="grid-block.image" src="/assets/images/logos/cardstack.svg" />
+            <img class="grid-block.image" src="/assets/images/logos/generali.svg" />
+          </div>
+          <p class="grid-block.text">
+            Generali approached simplabs when they were looking for support with an internal Ember.js project. We guided their team during the course of the project, taught and established best practices until the successful release.
```
Comment (mrmrcoleman) [inline src/ui/components/PageWork/template.hbs]: Generali approached simplabs when they were looking for support with an internal Ember.js project. We guided their team during the course of the project by teaching and establishing best practices until the project's successful completion.

File: src/ui/components/PageWork/template.hbs [hunk 4]
```diff
@@ -35,26 +35,106 @@
...
+          <p class="grid-block.text">
+            simplabs was approached by Clark when they were looking for guidance on how to solidify the codebase of their insurance management app. We conducted a thorough review, presented the workshop to the team and layed out a plan for addressing the identified issues.
+          </p>
+        </div>
+        <div class="grid-block.item">
+          <div class="grid-block.image-wrapper">
+            <img class="grid-block.image" src="/assets/images/logos/xbav.svg" />
+          </div>
+          <p class="grid-block.text">
+            xbAV makes pensions accessible for employees, employers and agents. They approached simplabs when they were looking for support releasing a number of critical features. Our technology experts joined xbAV's internal team, increasing the available workforce and sharing their expertise.
```
Comment (mrmrcoleman) [inline src/ui/components/PageWork/template.hbs]: xbAV makes pensions accessible for employees, employers and agents. They approached simplabs when they were looking for support releasing a number of critical features. Our technology experts joined xbAV's internal team, increasing the available workforce while boosting their expertise.

File: src/ui/components/PageWork/template.hbs [hunk 5]
```diff
@@ -35,26 +35,106 @@
...
+          </p>
+        </div>
+        <div class="grid-block.item">
+          <div class="grid-block.image-wrapper">
+            <img class="grid-block.image" src="/assets/images/logos/experteer.svg" />
+          </div>
+          <p class="grid-block.text">
+            Experteers is Europe's leading executive career service. simplabs has supported them in various ways ov the years, building custom web apps, reviewing their code as well as providing architecture and process consulting.
```
Comment (mrmrcoleman) [inline src/ui/components/PageWork/template.hbs]: Experteer is Europe's leading executive career service. simplabs has supported them in various ways over the years, building custom web apps, reviewing their code as well as providing architecture and process consulting.

File: src/ui/components/PageWork/template.hbs [hunk 6]
```diff
@@ -35,26 +35,106 @@
...
+          </p>
+        </div>
+        <div class="grid-block.item">
+          <div class="grid-block.image-wrapper">
+            <img class="grid-block.image" src="/assets/images/logos/mvb.svg" />
+          </div>
+          <p class="grid-block.text">
+            mvb provides marketing solutions for the german book industry. They approached simplabs when they were looking for external expertise. simplabs performed a workshop, leveling up the team's expertise and escorted the project until its successful completion.
```
Comment (mrmrcoleman) [inline src/ui/components/PageWork/template.hbs]: mvb provides marketing solutions for the german book industry. They approached simplabs when they were looking for external expertise. simplabs performed a workshop, leveling up the team's expertise and guided the project until its successful completion.

File: src/ui/components/PageWork/template.hbs [hunk 7]
```diff
@@ -35,26 +35,106 @@
...
+            LoCoNET builds online games for the web. They reached out to simplabs to improve their team's productivity. We identified and fixed some blocking issues, laid the architectural foundation for some advanced features and conducted workshops to level up their team's expertise.
+          </p>
+        </div>
+        <div class="grid-block.item">
+          <div class="grid-block.image-wrapper">
+            <img class="grid-block.image" src="/assets/images/logos/dim3.svg" />
           </div>
           <p class="grid-block.text">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Dim3 delivers software allowing hospitals manage and optimize their patients' nutrition plans. They approached simplabs when they faced severe performance problems in their app. We identified and fixed these issues and also some others along the way.
```
Comment (mrmrcoleman) [inline src/ui/components/PageWork/template.hbs]: Dim3 delivers software allowing hospitals to manage and optimize their patients' nutrition plans. They approached simplabs when they faced severe performance problem in their app. We identified and fixed these issues and also found and solved some potential future problems along the way.

**Review by mrmrcoleman — CHANGES_REQUESTED** [review]
Minor typos and English tweaks.

**Comment by marcoow** [comment]
@pichfl if you applied Mark's requested changes, feel free to approve and merge

**Review by pichfl — APPROVED** [review]

**Pull request merged**